### ISSUE TITLE
Why is focus trap disabled when action has a value?

### ISSUE THREAD

**Issue by nelsonwong1012** [elrumordelaluz/reactour#212]
Just wondering why this is and if there's a workaround for this. Thanks.

**Comment by elrumordelaluz** [comment]
Hi @nelsonwong1012 and thanks for the head up.
I am trying to remember when reviewing the `code`, but maybe @frassinier has a clean idea, since he made the original PR.

Thanks in advance to both of you!

**Comment by frassinier** [comment]
@nelsonwong1012 Do you mean [this](https://github.com/elrumordelaluz/reactour/pull/126/files?utf8=%E2%9C%93&diff=unified&w=1#diff-04c6e90faac2675aa89e2176d2eec7d8R353)?

If so, it's because you can point to any DOM element and let the user navigating.
It can be the case for the 10th step of the [demo](https://reactour.js.org/), you can programmatically give the focus to the "Show extra Buildings" button. It's not done here atm, but you can. :)

**Comment by elrumordelaluz** [comment]
Thanks @frassinier !!

Could #209 #129 and #204 related on it?

**Comment by frassinier** [comment]
Yes, they look related, indeed

**Comment by nelsonwong1012** [comment]
Interestingly, when I have `action` for a step, focus trap doesn't work. But if I use `actionBefore` instead, focus trap works!

**Comment by elrumordelaluz** [comment]
@nelsonwong1012 are you talking about `master` branch, right?

**Comment by nelsonwong1012** [comment]
@elrumordelaluz This is in my project where I'm using the latest released version on npm (1.16.0)

**Comment by nelsonwong1012** [comment]
Though, I still need to use `action` sometimes instead of `actionBefore` so would still like to see this fixed.

**Comment by elrumordelaluz** [comment]
There isn't `actionBefore` in [v1.16.0 released](https://unpkg.com/reactour@1.16.0/dist/reactour.cjs.js). There is only [action](https://github.com/elrumordelaluz/reactour/tree/v1#steps-example)

**Comment by elrumordelaluz** [comment]
Due to inactivity I am closing this _issue_, but feel free to re-open or create a new one in any case.
